common-kafka/src/main/java/com/cerner/common/kafka/consumer/ProcessingKafkaConsumer.java

```diff
@@ -194,6 +195,21 @@ public ProcessingKafkaConsumer(ProcessingConfig config) {
         this(config, new KafkaConsumer<>(config.getProperties()));
     }
 
+    /**
+     * Creates a new {@link ProcessingKafkaConsumer} with the given configuration and deserializers using the {@link KafkaConsumer}.
+     *
+     * @param config the configuration used by the consumer
+     * @param keyDeserializer The deserializer instance for value. When passing the deserializer instance
```

Comment:
`value` -> `key`

Reply:
yes, please change that

Reply:
https://github.com/cerner/common-kafka/pull/8/commits/3a78d8aabee1bb72a6a7793fa347eb6b09430fa1